app/books/[bookId]/page.tsx

```diff
@@ -0,0 +1,132 @@
+"use client";
+import { BOOKS_QUERY } from "@/lib/gql/book";
+import { REVIEWS_QUERY } from "@/lib/gql/review";
+import { Review } from "@/lib/types/review";
+import { useQuery } from "@apollo/client";
+import Link from "next/link";
+
+function StarRating({ rating }: { rating: number }) {
```

Comment:
1. Should be in a separate file
2. Should be reused in `app/books/[bookId]/review/[reviewId]/page.tsx`

Reply:
Fixed